skimage/registration/tests/test_masked_phase_cross_correlation.py

```diff
@@ -8,13 +8,23 @@
 from skimage._shared.testing import fetch, expected_warnings
 from skimage._shared.utils import _supported_float_type
 from skimage.data import camera, stereo_motorcycle
+from skimage.feature import masked_register_translation as _deprecated
 from skimage.io import imread
 from skimage.registration._masked_phase_cross_correlation import (
     _masked_phase_cross_correlation as masked_register_translation,
     cross_correlate_masked)
 from skimage.registration import phase_cross_correlation
 
 
+def test_detrecated_masked_register_translation():

```

Comment:
```suggestion
def test_deprecated_masked_register_translation():
```
Interesting... but is this related to the PR? I would expect this change to live (at least) in a separate commit!

Reply:
Good catch! No idea how that got deleted!  😅